third_party/taxonomy/data_catalog_schema.json

```diff
@@ -0,0 +1,103 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "$id": "https://github.com/IBM/the-mesh-for-data/pkg/taxonomy/data_catalog_schema.json",
+    "title": "Data Catalog Taxonomy",
+    "description": "Portion of the taxonomy owned by the data catalog.",
+    "definitions": {
+        "data_sensitivity": {
+            "type": "string",
+            "description": "Level of sensitivity of the data.",
+            "enum": ["confidential", "personal", "special personal"]
+        },
+        "geography_name": {
+            "type": "string",
+            "description": "Name of the geography",
+            "enum": ["Netherlands", "Turkey"]                              
+        },
+        "geography": {
+            "type": "object",
+            "properties": {
+                "geography_type": {
+                    "type": "string",
+                    "description": "Geographical resolution",
+                    "enum": ["union", "country", "state", "city", "region"]
+                },
+                "name": {
+                    "type": "string",
+                    "$ref": "#/definitions/geography_name"                      
+                }
+            }
+        },
+        "data_format": {
+            "type": "string",
+            "description": "Format in which the data is stored.",
+            "enum": ["parquet", "csv", "arrow", "json", "table", "avro", "binary"]
+        },
+        "data_type": {
+            "type": "string",
+            "description": "Type of data.",
+            "enum": ["tabular", "image", "text", "voice", "video"]
+        },
+        "metadata": {
+            "type": "array",
+            "description": "A list of single item or key value pairs.  Values must be one of the items indicated.",
+            "items": {
+                "oneOf":[
+                    {
+                        "type":"string",
+                        "oneOf": [ {"$ref": "#/definitions/data_sensitivity"}, {"$ref": "#/definitions/geography_name"} ]
+                    },
+                    {
+                        "type":"object",
+                        "patternProperties": {
+                            "^.*$":{
+                                "oneOf": [ {"$ref": "#/definitions/data_sensitivity"}, {"$ref": "#/definitions/geography_name"} ]
+                            }
+                        }
+                    }
+                ]
+            }
+        },
+        "resource": {
+            "type": "object",
+            "description": "Data set or other type of resource.",
+            "properties": {
+                "name": {
+                    "type": "string",
+                    "description": "Unique name of the data set"
+                },
+                "category": { "type": "string", "$ref": "#/definitions/data_type"},
+                "location": { "type": "object", "$ref": "#/definitions/geography"},
+                "tags": { "$ref": "#/definitions/metadata"},
```

Comment:
why not have category and location as tags? i.e., dataset is a set of {name, tags, columns}
Tags already include data sensitivity and geography.

Reply:
done